test/cli/test_cli_functions.sh

```diff
@@ -100,14 +100,16 @@ function timeout {
 
 function install {
 # TODO: once we can control the core resource amount we can use it,
-#       for now we will mimic the install command 
+#       for now we will mimic the install command
     noobaa crd create
     noobaa system create
```

Comment:
We already support --core-resources and --db-resources and --endpoint-resources as flags on `noobaa system create` so we can remove those kubectl patches

Reply:
@nb-ohad just like you did for test-olm.sh

Reply:
Changed the test to use the new --mini flag in #183